packages/webpack-plugin/src/plugin-utils.ts

```diff
@@ -364,8 +364,12 @@ export function getSortedModules(stylableModules: Map<NormalModule, BuildData |
 
 export function reportNamespaceCollision(
     namespaceToFileMapping: Map<string, Set<NormalModule>>,
-    compilation: Compilation
+    compilation: Compilation,
+    ignore?: boolean | 'warn'
```

Comment:
I suggest simplifying this API to contain less types, probably a string enum normalized on entry